src/demux/adts.ts

```diff
@@ -4,14 +4,17 @@
  */
 import { logger } from '../utils/logger';
 import { ErrorTypes, ErrorDetails } from '../errors';
+import {
+  DemuxedAudioTrack
+} from '../types/demuxer';
 
-import { Events } from '../events';
+import { Events, HlsEventEmitter } from '../events';
 
-export function getAudioConfig (observer, data, offset, audioCodec) {
-  let adtsObjectType; // :int
-  let adtsExtensionSampleingIndex; // :int
-  let adtsChanelConfig; // :int
-  let config;
+export function getAudioConfig (observer, data: Uint8Array, offset: number, audioCodec: string) {
```

Comment:
Is there a reason `observer` is not typed?

```suggestion
export function getAudioConfig (observer: HlsEventEmitter, data: Uint8Array, offset: number, audioCodec: string) {
```

Reply:
Property 'trigger' does not exist on type 'HlsEventEmitter'. so I removed type of `observer`.